extensions/wikia/AdEngine/js/AdEngineRunner.js

```diff
@@ -19,9 +19,11 @@ define('ext.wikia.adEngine.adEngineRunner', [
 	 * @param {function} runAdEngine
 	 */
 	function delayRun(runAdEngine) {
-		var enabledBidders = [],
-			biddersQueue = [],
-			startedByBidders = false;
+		var biddersQueue = [],
+			enabledBidderNames = [],
```

Comment:
I'd change implementation of `diff` method to make more magic based on existing `enabledBidders` array